removals/mp_gain_year_count_all_forest_types.py

```diff
@@ -174,7 +174,7 @@ def mp_gain_year_count_all_forest_types(tile_id_list):
 
         # Combines the four above gain year count tiles for each Hansen tile into a single output tile
         if cn.count == 96:
-            processes = 84   # 28 processors = 220 GB peak; 62 = 470 GB peak; 78 = 600 GB peak; 80 = 620 GB peak; 84 = 630 GB peak
+            processes = 50   # 28 processors = 220 GB peak; 62 = 470 GB peak; 78 = 600 GB peak; 80 = 620 GB peak; 84 = 630 GB peak
```

Comment:
Replace the comment with 50 = XXX GB peak.

Reply:
Changes made in the following commit: bcf63c1c802f8e9327579dfffe90b2654b52ff53